cocos/renderer/CCRenderer.cpp

```diff
@@ -321,10 +321,8 @@ void Renderer::addCommand(RenderCommand* command, int renderQueue)
     CCASSERT(!_isRendering, "Cannot add command while rendering");
     CCASSERT(renderQueue >=0, "Invalid render queue");
     CCASSERT(command->getType() != RenderCommand::Type::UNKNOWN_COMMAND, "Invalid Command Type");
-    if (command->isTransparent())
-        _transparentRenderGroups.push_back(command);
-    else
-        _renderGroups[renderQueue].push_back(command);
+    
+    _renderGroups[renderQueue].push_back(command);
```

Comment:
@darkdukey 
@minggo 
The global z in transparent objects is different with that in opaque objects.
The global z in transparent objects is the distance to the camera. You will get wrong result sort the transparent and opaque objects together.